README.rst

```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
 ::
 
 $ python host_os.py build-iso --help
+
+How to contribute a new package to Host OS
+------------------------------------------
+
```

Comment:
As we talked personally today, simply point to documentation in the versions git repository from here and add this text there.